extension/chrome/elements/compose-modules/compose-quote-module.ts

```diff
@@ -177,6 +187,9 @@ export class ComposeQuoteModule extends ViewModule<ComposeView> {
     if (decryptRes.success) {
       return decryptRes.content.toUtfStr();
     } else if (decryptRes.error && decryptRes.error.type === DecryptErrTypes.needPassphrase) {
+      if (Catch.isThunderbirdMail() && this.view.useFullScreenSecureCompose) {
+        await Time.sleep(2300);
+      }
```

Comment:
what's usage for this `time.sleep()`, is it needed?
I tried to delete it and didn't notice any issues

Reply:
Time.sleep() seems to fix the issue where "decrypting..." phase stucks for decrypting quoted email when replying to encrypted email. Background scripts seems to comunicate to the page but the page is not ready yet.

Reply:
Seems like some other change fixed this issue, as decrypting works well for me even after removing this `Time.sleep` part. Please check on your side

Reply:
Ok, will check tomorrow morning.

Reply:
Hi @sosnovsky - Thanks for raising this. Upon checking, the issue still exists on my end after removing this grace period allowance `Time.sleep()`. For example, the issue will reproduce on the recent encrypted conversion. I think this is just bare minimum, otherwise, it will be stuck rendering "decrypting..." with no way the passphrase dialog not appearing.

Reply:
Got it 👍